app/controllers/test/CustomsDeclarationsStubController.scala

```diff
@@ -16,20 +16,21 @@
 
 package controllers.test
 
+import config.AppConfig
 import javax.inject.{Inject, Singleton}
 import models.Field._
 import models._
-import play.api.http.ContentTypes
-import play.api.mvc.{Action, MessagesControllerComponents}
-import services.NotificationService
-import uk.gov.hmrc.http.HeaderCarrier
+import play.api.http.{ContentTypes, HeaderNames}
+import play.api.mvc.{Action, Codec, MessagesControllerComponents}
+import uk.gov.hmrc.http.{HeaderCarrier, HttpResponse}
 import uk.gov.hmrc.play.bootstrap.controller.FrontendController
+import uk.gov.hmrc.play.bootstrap.http.HttpClient
 
 import scala.concurrent.ExecutionContext
 import scala.xml._
 
 @Singleton
-class CustomsDeclarationsStubController @Inject()(notificationService: NotificationService, mcc: MessagesControllerComponents)(
+class CustomsDeclarationsStubController @Inject()(appConfig: AppConfig, httpClient: HttpClient, mcc: MessagesControllerComponents)(
```

Comment:
Just inject the bit of config you need?  cdsFileUpload?

Reply:
That's a great idea, I'll try it

Reply:
But it's not possible or something is missing. Here there is a pureconfig library for config, so it looks slightly different.